<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VCWind</class>
 <widget class="QMainWindow" name="VCWind">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>475</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QTabWidget" name="tabWidget">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <widget class="QWidget" name="tab_resource">
         <attribute name="title">
          <string>基础属性</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,1,2,2">
          <item row="2" column="0">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2" rowspan="3">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>资源包</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="1" column="0">
              <widget class="QPushButton" name="pb_add_rp">
               <property name="text">
                <string>添加</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="pb_remove_rp">
               <property name="text">
                <string>删除</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QListWidget" name="lw_rp">
               <property name="dragEnabled">
                <bool>true</bool>
               </property>
               <property name="dragDropMode">
                <enum>QAbstractItemView::InternalMove</enum>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::ExtendedSelection</enum>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <property name="movement">
                <enum>QListView::Snap</enum>
               </property>
               <property name="layoutMode">
                <enum>QListView::Batched</enum>
               </property>
               <property name="viewMode">
                <enum>QListView::ListMode</enum>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="3" rowspan="3">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>方块id文件（json）</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="1" column="1">
              <widget class="QPushButton" name="pb_remove_bsl">
               <property name="text">
                <string>删除</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pb_add_bsl">
               <property name="text">
                <string>添加</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QListWidget" name="lw_bsl">
               <property name="dragDropMode">
                <enum>QAbstractItemView::InternalMove</enum>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::ExtendedSelection</enum>
               </property>
               <property name="movement">
                <enum>QListView::Snap</enum>
               </property>
               <property name="layoutMode">
                <enum>QListView::Batched</enum>
               </property>
               <property name="viewMode">
                <enum>QListView::ListMode</enum>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>像素画方向</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="1" column="0">
              <widget class="QRadioButton" name="rdb_direction_side">
               <property name="text">
                <string>侧面（侧视）</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QRadioButton" name="rdb_direction_up">
               <property name="text">
                <string>上面（俯视）</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="rdb_direction_down">
               <property name="text">
                <string>下面（仰视）</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>游戏版本</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="2" column="1">
              <widget class="QRadioButton" name="rdb_version_15">
               <property name="text">
                <string>1.15</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QRadioButton" name="rdb_version_12">
               <property name="text">
                <string>1.12</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QRadioButton" name="rdb_version_16">
               <property name="text">
                <string>1.16</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QRadioButton" name="rdb_version_13">
               <property name="text">
                <string>1.13</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="rdb_version_14">
               <property name="text">
                <string>1.14</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QRadioButton" name="rdb_version_17">
               <property name="text">
                <string>1.17</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QRadioButton" name="rdb_version_19">
               <property name="text">
                <string>1.19</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QRadioButton" name="rdb_version_18">
               <property name="text">
                <string>1.18</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="sb_max_layers">
            <property name="prefix">
             <string>最大厚度：</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>3</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1" rowspan="2">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_allowed_blocks">
         <attribute name="title">
          <string>方块列表</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>全部方块</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="0" column="0">
              <widget class="QScrollArea" name="sa_blocks">
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>820</width>
                  <height>303</height>
                 </rect>
                </property>
                <layout class="QGridLayout" name="gl_sa_blocks"/>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>快捷键</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_16">
             <item row="2" column="0">
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="pb_select_all">
               <property name="text">
                <string>全部启用</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pb_deselect_all">
               <property name="text">
                <string>全部禁用</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_image">
         <attribute name="title">
          <string>选择图片</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6" columnstretch="0,0,0,0,0,0,0,0,0,0">
          <item row="0" column="8" rowspan="5">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="9" rowspan="4">
           <widget class="QLabel" name="lable_converted">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="cb_show_raw_size">
            <property name="text">
             <string>显示原始尺寸</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QToolButton" name="tb_add_images">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QToolButton" name="tb_remove_images">
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2" colspan="3">
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>调色算法</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_12">
             <item row="6" column="0">
              <widget class="QCheckBox" name="cb_algo_dither">
               <property name="text">
                <string>抖动</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="rb_algo_HSV">
               <property name="text">
                <string notr="true">HSV</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QRadioButton" name="rb_algo_RGB">
               <property name="text">
                <string notr="true">RGB</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QRadioButton" name="rb_algo_Lab00">
               <property name="text">
                <string notr="true">Lab00(CIEDE00)</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QRadioButton" name="rb_algo_RGB_Better">
               <property name="text">
                <string notr="true">RGB+</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QRadioButton" name="rb_algo_Lab94">
               <property name="text">
                <string notr="true">Lab94(CIEDE94)</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QRadioButton" name="rb_algo_XYZ">
               <property name="text">
                <string notr="true">XYZ</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0" rowspan="2" colspan="3">
           <widget class="QListWidget" name="lw_image_files">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="dragEnabled">
             <bool>true</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::InternalMove</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::ExtendedSelection</enum>
            </property>
            <property name="movement">
             <enum>QListView::Snap</enum>
            </property>
            <property name="isWrapping" stdset="0">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="4" rowspan="4">
           <widget class="QLabel" name="label_raw_image">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QCheckBox" name="cb_show_raw">
            <property name="text">
             <string>显示原图</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="9">
           <widget class="QCheckBox" name="cb_show_converted">
            <property name="text">
             <string>显示转化后图像</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_export">
         <attribute name="title">
          <string>导出</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7"/>
        </widget>
        <widget class="QWidget" name="tab_gpu">
         <attribute name="title">
          <string>显卡设置</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_8">
            <property name="title">
             <string>OpenCL 可用的计算设备</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_14">
             <item row="0" column="2">
              <widget class="QTreeWidget" name="tw_gpu">
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectItems</enum>
               </property>
               <property name="rootIsDecorated">
                <bool>true</bool>
               </property>
               <attribute name="headerCascadingSectionResizes">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string notr="true">CPU &amp; GPU</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_9">
            <property name="title">
             <string>计算设置</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_15">
             <item row="2" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0">
              <widget class="QSpinBox" name="sb_threads">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="prefix">
                <string>CPU线程数：</string>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>1024</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QComboBox" name="combobox_select_device"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_tests">
    <property name="title">
     <string>Tests</string>
    </property>
    <addaction name="action_test01"/>
   </widget>
   <addaction name="menu_tests"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_test01">
   <property name="text">
    <string>test01</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
